nemo/collections/asr/modules/flashlight_decoder.py

```diff
@@ -133,9 +148,11 @@ def __init__(
         self.vocab_size = self.tokenizer_wrapper.vocab_size
         self.blank = self.tokenizer_wrapper.blank
         self.silence = self.tokenizer_wrapper.unk_id
-
         if lexicon_path is not None:
-            self.lexicon = load_words(lexicon_path)
+            if lexicon_path == "DEFAULT_TOKEN_OFFSET":
```

Comment:
Where is it documented?

Reply:
added
https://github.com/NVIDIA/NeMo/blob/karpnv/beamsearch/docs/source/asr/asr_language_modeling_and_customization.rst?plain=1#L228